py-polars/tests/unit/test_series.py

```diff
@@ -2055,11 +2055,6 @@ def test_duration_arithmetic() -> None:
         assert_series_equal(df1["d_offset"], df2["d_offset"])
 
 
-def test_mean_overflow() -> None:
-    arr = np.array([255] * (1 << 17), dtype="int16")
-    assert arr.mean() == 255.0
```

Comment:
This doesn't test anything Polars-related, so I removed it.

Reply:
It does! It was testing if it doesn't overflow the mean. A naive sum/count would overflow the i16

Reply:
Hmm, but it's just `numpy` code though? 🤔 We don't rely on this to calculate our mean, right?

Reverted this removal, but I still don't understand why this test is in there!